### PR TITLE
[Bullet] Adds missing getters/setters revolute joints

### PR DESCRIPTION
**Pull request by Blast545**
This PR:

* Adds revoluteJoints with world as parent
* Fixes segfault related to default axis
* Adds missing getters/setters for revoluteJoints

This PR was tested using `joint_controller.sdf` and `joint_position_controller.sdf`

### CODE REVIEW

File: bullet/src/SDFFeatures.cc [hunk 1]
```diff
@@ -332,32 +276,38 @@ Identity SDFFeatures::ConstructSdfJoint(
...
-  btHingeConstraint* joint = new btHingeConstraint(
-    *this->links.at(childId)->link,
-    *this->links.at(parentId)->link,
-    convertVec(ignition::math::eigen3::convert(pivotChild)),
-    convertVec(ignition::math::eigen3::convert(pivotParent)),
-    convertVec(ignition::math::eigen3::convert(axisChild)),
-    convertVec(ignition::math::eigen3::convert(axisParent)));
+  btHingeAccumulatedAngleConstraint* joint;
+  if (parentId != worldId) {
+    joint = new btHingeAccumulatedAngleConstraint(
```
Comment (Lobotuerk) [inline bullet/src/SDFFeatures.cc:297]: Do this really need to be created as a `btHingeAccumulatedAngleConstraint` or can it be created as a `btHingeConstraint` and then casted?

Comment (Blast545) [reply bullet/src/SDFFeatures.cc:297]: I didn't try setting it to btHingeConstraint, as we are using getHingeAccumulatedAngle in the position getters. Is it there a good reason I may be missing to create it as btHingeConstraint?

Comment (Lobotuerk) [reply bullet/src/SDFFeatures.cc:297]: It was a `btHingeConstraint* joint = new btHingeConstraint(` before you changed it, I'm asking the reason behind the change

Comment (Blast545) [reply bullet/src/SDFFeatures.cc:297]: Sure. In the documentation `getHingeAccumulatedAngle` is only available for `btHingeAccumulatedConstraint`, it's weird the way it was before, and probably the angle returned wasn't correct.
 https://pybullet.org/Bullet/BulletFull/classbtHingeAccumulatedAngleConstraint.html#a88d009b1a4c8ba6ad95790358c5279bf

Comment (Lobotuerk) [reply bullet/src/SDFFeatures.cc:297]: Ok, makes sense

**Review by Lobotuerk — APPROVED** [review]

**Pull request merged**